packages/catalog-realm/fields/color-field/components/advanced-edit.gts

```diff
@@ -360,7 +360,12 @@ export default class AdvancedEdit extends Component<ColorFieldSignature> {
   handleFormatSelect(option: { label: string; value: RichColorFormat } | null) {
```

Comment:
can we call all components inside color-field/components without "-edit". And treat them like plain glimmer components

For example, color-field-edit -> color-picker and advanced-edit -> advanced-color-picker